tests/tasks/setup-mock-wifi.yml

```diff
@@ -0,0 +1,91 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Please avoid dashes in the filename

Reply:
Have made a PR to remove the dashes from all test filenames here  #203

Reply:
fixed